snippets/custom_columns.js

```diff
@@ -0,0 +1,79 @@
+var msa = window.msa;
+
+yourDiv.textContent = "loading";
+
+var opts = {};
+opts.el = yourDiv;
+
+const customValueCalculator = (id) => {
```

Comment:
nit: The name `calculator` kinda implies that it'll calculate a different custom value for each ID.
How about renaming to `getCustomText`.